src/foam/java/refinements.js

```diff
@@ -55,6 +55,7 @@ foam.CLASS({
         propName: this.name,
         propType: this.javaType,
         propRequired: this.required,
+        of: this.of,
```

Comment:
Change to: this.of.id, since you just want to store the package and name of the 'of', not the JS Object itself, which doesn't make any sense to Java.